experimental/add_to_app/flutter_module/lib/main.dart

```diff
@@ -72,8 +72,8 @@ class _MiniViewState extends State<MiniView> {
               mainAxisAlignment: MainAxisAlignment.center,
```

Comment:
Random extra comment (for line 35 but I can't add a comment to a line that far up): Why is this called MiniView? It doesn't actually do anything special relating to that name right? It's just the body.

Reply:
Yeah, that's a good point. The names were left over from a slightly different version earlier. I've changed "MiniView" to "Contents."